src/src/AdminBundle/Controller/AdminController.php

```diff
@@ -32,36 +35,94 @@ public function hashPassword($admin, $password) {
      * @Route("/admins/create", methods={"POST"})
      */
     public function createAdmin(Request $request) {
-        $email = $request->get('email');
-        $password = $request->get('password');
-        $password1 = $request->get('password1');
-
-        if ($password == "" or $password1 == "") {
-            $error = "Enter password 2x, please";
-            return $this->render('AdminBundle:Admin:create.html.twig', array(
-                'error' => $error
-            ));
-        }
+        $defaultData = array(
+            'email' => '',
+            'password' => '',
+            'password1' => ''
+        );
 
-        if ($password != $password1) {
-            $error = "Passwords do not match";
-            return $this->render('AdminBundle:Admin:create.html.twig', array(
-                'error' => $error
-            ));
-        }
+        $form = $this->createFormBuilder($defaultData)
+            ->add('email', EmailType::class)
+            ->add('password', RepeatedType::class, array(
+                'type' => PasswordType::class,
+                'invalid_message' => 'The password fields must match.',
+                'options' => array('attr' => array('class' => 'password-field')),
+                'required' => true,
+                'first_options'  => array('label' => 'Password'),
+                'second_options' => array('label' => 'Repeat Password'),
+            ))
+            ->add('submit', SubmitType::class)
+            ->getForm();
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            // data is an array with, "email", and "password" keys
+            $data = $form->getData();
+            $email = $data['email'];
+            $password = $data['password'];
+
+            $admin = new Admin($email, $password);
```

Comment:
`new Admin($email, $password);` heslo sa tu posiela zbytocne, rado tuto entitu zmeni aby sa to v konstruktore neposielalo